### ISSUE TITLE
Windows wheel installed with pip cannot be loaded unless libpython is copied in their folder

### ISSUE THREAD

**Issue by touilleMan** [touilleMan/godot-python#228]
```
$ .\addons\pythonscript\windows-64\python.exe -m pip --version
pip 19.2.3 from C:\Users\gbleu\source\repos\godot-python\examples\pong\addons\pythonscript\windows-64\lib\site-packages\pip (python 3.8)

$ .\addons\pythonscript\windows-64\python.exe -m ensurepip
Looking in links: c:\Users\gbleu\AppData\Local\Temp\tmp11e9hncn
Requirement already satisfied: setuptools in c:\users\gbleu\source\repos\godot-python\build\dist\addons\pythonscript\windows-64\lib\site-packages (41.2.0)
Requirement already satisfied: pip in c:\users\gbleu\source\repos\godot-python\build\dist\addons\pythonscript\windows-64\lib\site-packages (19.2.3)

$ .\addons\pythonscript\windows-64\python.exe -m pip install tcod
Collecting tcod
  Downloading https://files.pythonhosted.org/packages/fb/37/de0f59a240f9042ec7232214926e9ff359f2ff6f890b46367d250324c49e/tcod-11.15.3-cp38-cp38-win_amd64.whl (1.2MB)
     |████████████████████████████████| 1.2MB 6.4MB/s
Collecting numpy~=1.10 (from tcod)
  Downloading https://files.pythonhosted.org/packages/c7/7d/ea9e28c3a99f50e77ee9a0e3759adb6537b2bb7a84aef27b8c0ddc431b48/numpy-1.19.1-cp38-cp38-win_amd64.whl (13.0MB)
     |████████████████████████████████| 13.0MB ...
Collecting cffi~=1.13 (from tcod)
  Downloading https://files.pythonhosted.org/packages/40/ad/eb98b5ec6129ffdabedca218ded2c529d59b935dac7cc6108366e379de96/cffi-1.14.1-cp38-cp38-win_amd64.whl (178kB)
     |████████████████████████████████| 184kB ...
Collecting pycparser (from cffi~=1.13->tcod)
  Using cached https://files.pythonhosted.org/packages/ae/e7/d9c3a176ca4b02024debf82342dab36efadfc5776f9c8db077e8f6e71821/pycparser-2.20-py2.py3-none-any.whl
Installing collected packages: numpy, pycparser, cffi, tcod
  WARNING: The script f2py.exe is installed in 'C:\Users\gbleu\source\repos\godot-python\examples\pong\addons\pythonscript\windows-64\Scripts' which is not on PATH.
  Consider adding this directory to PATH or, if you prefer to suppress this warning, use --no-warn-script-location.
Successfully installed cffi-1.14.1 numpy-1.19.1 pycparser-2.20 tcod-11.15.3
WARNING: You are using pip version 19.2.3, however version 20.2 is available.
You should consider upgrading via the 'python -m pip install --upgrade pip' command.

$ ls .\addons\pythonscript\windows-64\Lib\site-packages\tcod
__init__.py   _internal.py   bsp.py    console.py    context.py  event_constants.py  libtcodpy.py  los.py  noise.py  py.typed   sdl.py      version.py
__pycache__/  _libtcod.pyd*  color.py  constants.py  event.py    image.py            loader.py     map.py  path.py   random.py  tileset.py  x64/

$ .\addons\pythonscript\windows-64\python.exe
Python 3.8.3 (default, May 17 2020, 15:15:52) [MSC v.1925 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import tcod
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\gbleu\source\repos\godot-python\examples\pong\addons\pythonscript\windows-64\lib\site-packages\tcod\__init__.py", line 18, in <module>
    from tcod.loader import lib, ffi, __sdl_version__  # noqa: F4
  File "C:\Users\gbleu\source\repos\godot-python\examples\pong\addons\pythonscript\windows-64\lib\site-packages\tcod\loader.py", line 111, in <module>
    from tcod._libtcod import lib, ffi  # type: ignore # noqa: F401
ImportError: DLL load failed while importing _libtcod: The specified module could not be found.
>>> exit()
```

According to @cridenour, copying the `python3.dll` in the `addons\pythonscript\windows-64\lib\site-packages\tcod` folder solve the issue (see https://youtu.be/dGmV9U095Qs?t=155).

My guess is the issue comes from the fact there is no `python3.dll` in the `addons\pythonscript\windows-64\` folder (it should be there along with `python38.dll`). I'm going to create an issue on the python-build-standalone project about this ;-)

@cridenour can you try to copy `python3.dll` in `addons\pythonscript\windows-64\` to make sure it fixes the issue ?

**Comment by cridenour** [comment]
I can confirm having `python3.dll` only in `windows-64` also works.

**Comment by touilleMan** [comment]
should have been fixed by 7a75e8bed2d429396b099e36f546660a800c5b5c
